docs/sql/data_types.txt

```diff
@@ -268,6 +268,15 @@ Both of these index types accept the following parameters:
                      e.g. points that are not exactly inside this shape will end up inside it
                      when it comes to querying as the shape has grown when indexed.
 
+:tree_levels: Maximum number of layers to be used by the ``PrefixTree`` defined
+              by the index type (either ``geohash`` or ``quadtree``). This can be used to control
```

Comment:
maybe add a reference to "geo shape index structure" below for more information about internals.